protosocket-connection/src/connection.rs

```diff
@@ -393,7 +426,10 @@ where
         Poll::Pending
     }
 
-    /// send buffers to the tcp stream, and recycle them if they are fully written
+    /// Send buffers to the tcp stream, and recycle them if they are fully written
+    /// Prepares outbound messages for sending by serializing them and
+    /// moving them to the write queue. Handles cases where the outbound
+    /// message channel is empty or closed.
```

Comment:
This does not serialize messages, and it does not move messages to the send queue.

Reply:
yeah this a copy pasta error on my side 🤦